en/micro-integrator/docs/references/connectors/exporting-artifacts.md

```diff
@@ -0,0 +1,27 @@
+# Exporting artifacts
```

Comment:
We cannot use this heading number if we are inserting it into another page. Probably it will be ##.

Reply:
Done